static/js/encryption.js

```
@@ -0,0 +1,99 @@
+      ["encrypt", "decrypt"]
+    );
+  }
+
+  /**
+   * Encrypts a message
+   * @param {string} message - Plaintext message
+   * @param {string} keyBase64 - Base64-encoded key
+   * @returns {Promise<string>} Base64-encoded encrypted message (IV + ciphertext)
+   */
+  static async encryptMessage(message, keyBase64) {
+    const key = await this.importKey(keyBase64);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)